src/styles/_catalog.scss

```diff
@@ -1,10 +1,27 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  padding: 20px;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 1fr);
+  }
```

Comment:
You can put this template columns in .catalog and delete this @media, but i think it's just cosmetics.